config/src/config/secure_config.rs

```diff
@@ -3,14 +3,40 @@
 
 use serde::{Deserialize, Serialize};
 use std::path::PathBuf;
+use std::net::SocketAddr;
+
+const DEFAULT_JSON_RPC_ADDR :&str = "127.0.0.1";
+const DEFAULT_JSON_RPC_PORT: u16 = 8080;
 
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 #[serde(default, deny_unknown_fields)]
-pub struct SecureConfig {}
+pub struct SecureConfig {
+    pub key_manager: KeyManagerConfig,
+}
 
 impl Default for SecureConfig {
     fn default() -> Self {
-        Self {}
+        Self {
+            key_manager : KeyManagerConfig::default(),
+        }
+    }
+}
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct KeyManagerConfig {
```

Comment:
One might argue that the better approach may very well be to have specific configurations for each identity. So SafetyRules might have its own top level config that imports the various fields it cares about. I don't know which one is easier to test (monolithic like we have it) or specific. I suspect monolithic is easier to build and test but harder to reason about.

Reply:
Good point. As discussed offline, I'll look into this in a future PR